docs/release-notes/2.42.md

```diff
@@ -0,0 +1,104 @@
+# Percona Monitoring and Management 2.42
+
+| **Release date** | May 30th, 2024                                                                                   |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+It enables you to observe the health of your database systems, explore new patterns in their behavior, troubleshoot them and execute database management operations—regardless of whether your databases are located on-premises or in the cloud.
+
+## Release highlights
+
+## Ubuntu 24.04 Noble support
+
+PMM now officially supports monitoring of databases running on the recently released Ubuntu 24.04 Noble operating system.
+
+With this addition, PMM continues to provide comprehensive coverage across a wide range of Linux distributions commonly used in database deployments. For more information, see [blogpost]().
+
+## Configurable metrics resolutions per service
+
+You can now configure metrics resolutions on a per-service basis by setting Low-Resolution (LR), Medium-Resolution (MR), and High-Resolution (HR) settings for each exporter individually.
+
+Customizing the resolution settings for individual services means that you to fine-tune your PMM setup to balance data granularity and resource consumption.  This will enable you to:
+
+- allocate resources efficiently by focusing on high-resolution data for key services
+- reduce storage requirements by adjusting resolution for less critical components
+- align your monitoring setup with the specific needs of your environment
+  
+This feature is currently accessible via the PMM API and will be integrated into the user interface in a future release.
+
+## Configure maximum connections for PostgreSQL services
+
+You can now configure the maximum number of connections that the PMM Agent's actions/jobs runner can open to the same PostgreSQL instance.
+
+Previously available only through the CLI, this feature is now also accessible under:
+
+- PostgreSQL service: **PMM Configuration > Add Service > PostgreSQL > Configuring PostgreSQL Service**
+- RDS PostgreSQL service:  **PMM Configuration > Add Service > Amazon RDS > Discover with Amazon Credentials > RDS PostgreSQL**
+
+Setting a maximum connection limit helps prevent excessive connections during concurrent operations, such as running multiple EXPLAIN or other operations required by Query Analytics (QAN) and Advisors, ensuring connections are closed promptly and avoiding idle connections.
```

Comment:
This section is about jobs runner limits, while rest of the paragraph is about exporter. We have two different limits for connections, one on the exporter side and one on pmm-agent.

Reply:
Added suggestion below